tekton-catalog/pipeline-loops/pkg/apis/pipelineloop/v1alpha1/pipelineloop_defaults.go

```diff
@@ -31,4 +33,19 @@ func (tl *PipelineLoop) SetDefaults(ctx context.Context) {
 
 // SetDefaults set any defaults for the PipelineLoop spec
 func (tls *PipelineLoopSpec) SetDefaults(ctx context.Context) {
+	if tls.Parallelism == 0 {
```

Comment:
When I explicitly set Parallelism: 0 in my pipelineloop, it will still fall back to the default because this condition will pass for both null and 0. Shouldn't if a user specify `Parallelism: 0`, then the controller won't limit the number of parallel pipelineruns?

Reply:
Thanks Tommy，in fact I am intertwined with this default value too. The original design is to use 0 to represent unlimited. But you know in golang the default value for int is 0 which means if user did not specify it then it will be unlimited. Originally it's 1 without this feature so this maybe a surprise to user so I think we should fall back to default 1? BTW, in golang it's still 0 for unlimited and I will change it. What's your opinion?

Reply:
Addressed and readme updated based on our discussion, thanks.